ios/FluentUI/Tab Bar/TabBarItemView.swift

```diff
@@ -110,8 +110,8 @@ class TabBarItemView: UIView {
         if isInPortraitMode {
             container.axis = .vertical
             container.spacing = spacingVertical
-            imageHeightConstraint?.constant = portraitImageSize
-            imageWidthConstraint?.constant = portraitImageSize
+            imageHeightConstraint?.constant = titleLabel.isHidden ? portraitImageSize : landscapeImageSize
+            imageWidthConstraint?.constant = titleLabel.isHidden ? portraitImageSize : landscapeImageSize
```

Comment:
if we're using the landscapeImageSize, we should also use the item's landscapeImage. See below, when we're doing:

imageView.image = ...
imageView.highlightedImage = ...

This will involve perhaps modifying the unselected/selected Image methods to take another param, and also modifying the comments on the initializer of MSFTabBarItem, indicating that the landscapeImage will be used for portrait if the tab bar item shows a lablel